android/src/fdroid/play/listings/hu-HU/release-notes.txt

```diff
@@ -1,8 +1,10 @@
-• Új OpenStreetMap térképadatok (március 29)
-• Kísérleti sávnavigációs funkció
-• A felhasználó helyzetét jelző nyíl pozícióját megváltoztattuk, a jobb élmény érdekében
-• Megjavítottuk a könyvjelzők színválasztóját a jobb oldalról balra haladó írásmódú nyelveknél
-• Immáron a sebesség és a távolság helyesen van megjelenítve navigáció közben
-• Sok új bolttípussal bővült alkalmazásunk
-• Hozzáadtuk a pénz átutalóhelyeket, bicikli töltőállomásokat, és erőműveket
-…további infó: omaps.app/news
+• Új OpenStreetMap adatok (május 3)
+• A keresés most váltakozó páratlan és páros címeket talál
+• A metró réteg gyorsabb lett
+• Kijavítottuk a hiányzó Wikipedia linkeket
+• Javítottuk az autók körforgalmakon keresztüli navigációját
+• Javítottuk a helymeghatározás funkciót a hálózaton keresztül egyes eszközökön
+• Megváltozott a kapuk és a lyoni metró ikonja
+• Javítottuk az amerikai autópálya jelölő ikonokat
+• Kamerabolt, belső dekor, régiségek, művészetek, sajtbolt hozzáadása
+…infó: omaps.org/news
```

Comment:
"...további infó:" !!!! Not only "...infó:"! Please fix this ASAP ❗

Reply:
Google set a limit of max 500 characters for release notes. It would be great to size-check them next time. Now we can only update the description in Google Play.